### DISCUSSION TITLE
返回不到上次点击位置

### DISCUSSION THREAD

**Discussion by narutobclcc** [community/community#36361] [Mobile]
用安卓github阅读文章时，如果跳转到其他链接，再返回不会定位到上次阅读的位置，而且跳转到页面开始的地方

**Comment by candyho** [comment]
你好！请问你所说的跳转到其他链接，是外部链接还是链接至应用内的其他版面？

**Comment by narutobclcc** [comment]
链接至应用内的其他版面



---原始邮件---
发件人: "Candy ***@***.***&gt;
发送时间: 2022年10月19日(周三) 晚上8:38
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [community/community] 返回不到上次点击位置 (Discussion #36361)




 
你好！请问你所说的跳转到其他链接，是外部链接还是链接至应用内的其他版面？
